src/components/tabs/Best.tsx

```diff
@@ -0,0 +1,258 @@
+import { type FC, useEffect, useState } from "react";
```

Comment:
This file is very long - split up the components into smaller files for readability
https://medium.com/swlh/how-to-write-great-react-c4f23f2f3f4f#:~:text=50%20lines%20is%20a%20good,Under%20100%20is%20ideal.